lib/internal/process/execution.js

```diff
@@ -159,6 +162,7 @@ function createOnGlobalUncaughtException() {
     }
 
     const type = fromPromise ? 'unhandledRejection' : 'uncaughtException';
+    process.emit(kUncaughtExceptionMonitor, er, type);
```

Comment:
FWIW, this would not be invoked if the user monkey-patches `process._fatalException` without calling the original callback.

Reply:
I know, but I expect in that case also regular `uncaughtException` listeners at not called therefore we are lost already now.